hwcomposer/hwcomposer_backend_v11.cpp

```diff
@@ -72,12 +74,14 @@ struct HwcProcs_v11 : public hwc_procs
 
 static void hwc11_callback_vsync(const struct hwc_procs *procs, int, int64_t)
 {
+#ifdef WITH_SYSTRACE
```

Comment:
i'd suggest adding static functions wrapping the QSystrace use which can host the ifdef in a single place, rather than littering every function call with it

Reply:
We can also try it this way (pseudo-code):
```c++
#ifndef WITH_SYSTRACE
// Add dummy stub methods for QSysTrace
namespace QSystrace
{
    void begin(const char *module, const char *tracepoint, const char *message, ...) {}
    void end(const char *module, const char *tracepoint, const char *message, ...) {}
    void counter(const char *module, const char *tracepoint, const char *message, ...) {}
};
struct QSystraceEvent {
    QSystraceEvent(const char *module, const char *tracepoint) {}
};
#endif
```

Reply:
Yep, good idea :)